apps/src/lib/kits/maker/PlaygroundComponents.js

```diff
@@ -54,6 +54,64 @@ export function initializeCircuitPlaygroundComponents(board) {
 }
 
 /**
+ * De-initializes any Johnny-Five components that might have been created
+ * by createCircuitPlaygroundComponents
+ * @param {Object} components - map of components, as originally returned by
+ *   createCircuitPlaygroundComponents.  This object will be mutated: Destroyed
+ *   components will be removed. Additional members of this object will be
+ *   ignored.
+ */
+export function destroyCircuitPlaygroundComponents(components) {
+  if (components.colorLeds) {
+    components.colorLeds.forEach(led => led.stop());
+  }
+  delete components.colorLeds;
```

Comment:
I wondered about this, so I [wrote a test](https://github.com/code-dot-org/code-dot-org/pull/13638/files#diff-ca2026850a1bf45fc49a50c5fc166c80R366) that ensures the destroy method is safe to call on an empty object.